### DISCUSSION TITLE
[Help]: run in mikrotik hardware

### DISCUSSION THREAD

**Discussion by TheWickerman69** [mbentley/docker-omada-controller#422] [Help]
### Controller Version

v.5.7

### Describe Your Issue or Question

I'm struggling with several mikrotik routers and the possibility of running containers on them. I have tried with a L009UiGS-RM that has a dual-core 32-bit Arm processor and with an RB5009UG+S+IN that has a Marvell Armada Quad-core ARMv8 1.4 GHz CPU and I can't get either of them to start up. ...can someone shed some light for me?

### Expected Behavior

.can someone shed some light for me?

### Steps to Reproduce

1

### How You're Launching the Container

```plain
1
```

### Container Logs

```plain
log
```

### MongoDB Logs

```plain
logs
```

**Comment by mbentley** [comment]
It should start on a 32 bit ARM system but I wouldn't run it on one.  There are [significant issues as to why you should most definitely not do so](https://github.com/mbentley/docker-omada-controller/blob/master/KNOWN_ISSUES.md#notes-for-armv7l).  Technically it should work on 32 bit ARM as I still build images for it but is insecure as hell and the MongoDB is running an all around unsupported version.  Hard to say why it doesn't start with no information though.

That being said, a `RB5009UG+S+IN` has 64 bit ARM so if you can run a 64 bit OS, you'll be in a better spot.  1 GB of RAM isn't a lot though so there is also that.
